test/src/Server/ORM/CRUD/CRUDTest.php

```diff
@@ -525,12 +525,73 @@ function (DoctrineResourceEvent $e) {
         $this->validateTriggeredEvents(array());
     }
 
+
+    public function testDeleteList()
+    {
+        $serviceManager = $this->getApplication()->getServiceManager();
+        $em = $serviceManager->get('doctrine.entitymanager.orm_default');
+        $deleteList = array();
+
+        $this->getRequest()->getHeaders()->addHeaders(
+            array(
+                'Accept' => 'application/json',
+                'Content-type' => 'application/json',
+            )
+        );
+        $this->getRequest()->setMethod(Request::METHOD_POST);
+        $this->getRequest()->setContent('{"name": "ArtistOne","createdAt": "2011-12-18 13:17:17"}');
+        $this->dispatch('/test/artist');
+        $body = json_decode($this->getResponse()->getBody(), true);
+
+        $deleteList[] = array(
+            'id' => $body['id'],
+        );
+
+        $this->getRequest()->setMethod(Request::METHOD_POST);
+        $this->getRequest()->setContent('{"name": "ArtistTwo","createdAt": "2011-12-18 13:17:17"}');
+        $this->dispatch('/test/artist');
+        $body = json_decode($this->getResponse()->getBody(), true);
+
+        $deleteList[] = array(
+            'id' => $body['id'],
+        );
+
+        $this->getRequest()->setMethod(Request::METHOD_POST);
+        $this->getRequest()->setContent('{"name": "ArtistThree","createdAt": "2011-12-18 13:17:17"}');
+        $this->dispatch('/test/artist');
+        $body = json_decode($this->getResponse()->getBody(), true);
+
+        $deleteList[] = array(
+            'id' => $body['id'],
+        );
+
+        $em->clear();
+
+        $this->getRequest()->getHeaders()->addHeaders(
+            array(
+                'Accept' => 'application/json',
+                'Content-type' => 'application/json',
+            )
+        );
+        $this->getRequest()->setMethod(Request::METHOD_DELETE);
+        $this->getRequest()->setContent(json_encode($deleteList));
+        $this->dispatch('/test/artist');
+        $this->assertEquals(204, $this->getResponseStatusCode());
+
+        $this->validateTriggeredEventsContains(array(
+            DoctrineResourceEvent::EVENT_DELETE_PRE,
+            DoctrineResourceEvent::EVENT_DELETE_POST,
```

Comment:
The above two are not triggered by `deleteList()`, and are the source of the unit test failure. I've removed them locally for the merge review.